common/templates/header.tmpl

```diff
@@ -0,0 +1,16 @@
+{{define "header"}}
+//go:build go1.18
+// +build go1.18
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+package {{.Package}}_test
+
+import (
+	"testing"
+	"{{.Package}}"
+
```

Comment:
This is super nitpicky, but should the package under test be grouped with standard library imports?

```suggestion
	"testing"

	"{{.Package}}"
```

Reply:
I meant to remove these. I had them in place as I was trying to emit calling client constructors but couldn't get it to work.